crates/bevy_light/src/lib.rs

```diff
@@ -193,6 +219,20 @@ impl Plugin for LightPlugin {
                         .after(clear_directional_light_cascades),
                 ),
             );
+
+        if self.default_environment_map_light {
+            app.world_mut()
+                .register_required_components::<Camera3d, EnvironmentMapLight>();
+        }
+
+        app.world_mut().resource_mut::<Assets<Image>>().insert(
+            &DEFAULT_ENVIRONMENT_MAP_TEXTURE_HANDLE,
```

Comment:
Should the default environment light be pure white like this? I think a better default would be a sky-like gradient, or maybe even the atmosphere env light. probably out of scope for this PR though.